src/index.js

```diff
@@ -83,13 +85,15 @@ function getUserLanguage () {
 
 function flags () {
   const user = JSON.parse(window.localStorage.getItem(USER_KEY))
-  var flags_ = {
+  return {
     env: env,
+    graphqlSecret: graphqlSecret,
     endpoints: config.endpoints,
     language: getUserLanguage(),
     accountName: (user && user.accountName) || null,
     isPinAvailable: !!(user && user.encryptedKey),
     authPreference: window.localStorage.getItem(AUTH_PREF_KEY),
+    authToken: window.localStorage.getItem(AUTH_TOKEN),
```

Comment:
I'm a little bit confused. This `AUTH_TOKEN` is the token we get after calling `signUp` or `signIn`, or the password we send to those params?

Reply:
The [whole object](https://github.com/cambiatus/frontend/pull/485/files/e5eef2bbf8cadc930d6b140504fe1520c9975f5b#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R86) returned from `flags()` contains two properties related to authentication: `graphqlSecret` and `authToken`.

`graphqlSecret` is named after the config parameter [on the backend](https://github.com/cambiatus/backend/blob/44b7a3f8d9ac36bf50c467e051b574a5228f3702/config/dev.exs#L37
) and the `authToken` represents the response from the server with the token, generated by `CambiatusWeb.AuthToken.sign(user)` and could be stored/retrieved from the `localStorage`.

`graphqlSecred` is retrieved from the `GRAPHQL_SECRET` ENV variable during the build process.